css/properties/scroll-snap-type.json

```diff
@@ -21,24 +21,24 @@
               "prefix": "-ms-",
               "notes": "Edge supports an earlier draft of CSS Scroll Snap without axis values."
             },
-            "firefox": {
-              "version_added": "39",
-              "notes": "Firefox supports an earlier draft of CSS Scroll Snap without axis values."
-            },
+            "firefox": [
```

Comment:
We prefer blocks to be ordered in reverse chronological order.  Can we place the single `"version_added": "68"` as the first entry in the array?